### PR TITLE
Keeping ones cool as a consultant

### PR DESCRIPTION
**Pull request by marjampi**

### CODE REVIEW

File: _posts/2022-12-21-how-to-keep-your-cool-as-a-consultant.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+At least here at Solita we're offered plenty of possibilities to learn. The learning may happen in your daily project work or you can take time from your project to learn what you feel like is important to learn. We have internal courses which anyone can develop and share their knowledge about a subject. There's also links to external materials, like books in O'Reilly's platform. And the learning is not restricted to what the company "provides". We're free to pursue what interests us and at least I've not needed to explain to anyone why I've decided to use my time learning something.
+
+## What keeps me cool (or at least cooler...)
+
+At Solita, one of the most important things keeping me fit to work is my people lead. I know having an administrative manager is not ideal for everybody. But to me it is super important that there is this one person who cares about me and who I've built trust with over the years. I really can't see myself working in a company where you're supposed to contact an hr person and it might be a different person every time.
```
Comment (tatut) [inline _posts/2022-12-21-how-to-keep-your-cool-as-a-consultant.md:43]: should hr be capitalized into HR

File: _posts/2022-12-21-how-to-keep-your-cool-as-a-consultant.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Another thing helping me is Solita's culture. I feel it's okay not to be happy and content all the time. I can out my frustration freely both to other "normal" colleagues as well as to colleagues working in managerial positions. A few times I've talked with my project manager and just poured my heart out and told that everything is sh*t. And they've listened to me and after I've calmed down asked what concrete steps could be taken to make things easier.
+
+Also, feedback is asked from all Solitans regularly. I think it currently every three weeks that we get a link to a Peakon questionnaire. Many times in my answers I've told about smaller or bigger things bothering me. For me, a service like Peakon's is good, because the anonymity allows me to say about issues I might not raise if I had to say them with my face & name from the very beginning.
```
Comment (tatut) [inline _posts/2022-12-21-how-to-keep-your-cool-as-a-consultant.md:47]: use of ampersand to replace `and` is generally discouraged

**Review by tatut — COMMENTED** [review]
some general comments

File: _posts/2022-12-21-how-to-keep-your-cool-as-a-consultant.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+When rapid changes happen, it takes a while for me to adjust to the new situation. I wrote something along the lines of "I prefer stability over fast changes" to my cv in Solita's internal cv bank. A colleague who was proofreading it said that it sound like I prefer waterfall development over agile methods. I then realized my colleague and I thought of very different kind of changes. What I had had in my mind at the time of the writing were more along the lines of customer announcing "we're not going to been half you after this sprint, good bye", not if we should try a new kind of retro template this week or maybe try a different kind of approach with planning the sprint. In the small everyday situations I can be very flexible (as long as I have some familiar things), but those big "announcements" that take effect almost immediately are really not easy for me to handle.
```
[the remark this answers is not in the document]
Comment (marjampi) [reply _posts/2022-12-21-how-to-keep-your-cool-as-a-consultant.md:25]: 👀  yes I have a completely wrong word there

**Review by mattiuusitalo — APPROVED** [review]
Seems better balanced now with the latest revision! Great post.

**Pull request merged**